CHANGELOG.md

```diff
@@ -1,4 +1,14 @@
 # Generic-Parser Changelog
+## Version 1.0.7
+ * fixed crash before proper error message for invalid choice of on non-string
+ objects. 
+ * fixed `None` write and read in config files.
+ * added tests for list-lengths of default parameters.
+ * improved `create_parameter_help` function.
+ * some documentation fixed/additions.
+ * switched to github actions for CI.
```

Comment:
not no this branch?

Reply:
true, but it's the Version 1.0.7 changelog ... this is all in the same project

Reply:
Perfect